site/src/analytics-average.ts

```diff
@@ -29,8 +29,9 @@ export class AnalyticsAverage extends LitElement {
         </div>
       </div>
       <div class="footer">
-        ${this.lastDataEntry!.reports_statistics || "Unkown"} installations are
-        currently reporting statistics
+        ${this.lastDataEntry!.reports_statistics || "Unkown"} of
```

Comment:
How come you need a `!` here but not in the next line?

Reply:
It was not needed at all